0.3.3 release doesn't build out of the box by unittests build

master is also affected by this issue:
```
sdbus-cpp/test/unittests/Types_test.cpp: In member function ‘virtual void AVariant_CanBeConstructedFromAComplexValue_Test::TestBody()’:
sdbus-cpp/test/unittests/Types_test.cpp:69:35: error: unnecessary parentheses in declaration of ‘value’ [-Werror=parentheses]
     ASSERT_NO_THROW(sdbus::Variant(value));
                                   ^
```

Possible fix there: merhalak@93248b80a2ea1f418f88e545fe933532c690a515

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -65,7 +65,6 @@ set(SDBUSCPP_SRCS ${SDBUSCPP_CPP_SRCS} ${SDBUSCPP_HDR_SRCS} ${SDBUSCPP_PUBLIC_HD
 #-------------------------------
 
 set(CMAKE_CXX_STANDARD 17)
-add_compile_options(-W -Wextra -Wall -Werror -pedantic)
 include_directories("${CMAKE_SOURCE_DIR}/include")
 include_directories("${CMAKE_SOURCE_DIR}/src")
 

